src/components/Questions.tsx

```diff
@@ -53,7 +41,10 @@ export default function Questions() {
         </ListSubheader>
       }
     >
-      <ListItemButton onClick={handleClick}>
+      <ListItemButton
+        selected={selectedIndex === 0}
+        onClick={(event) => handleListItemClick(event, 0)}
+      >
         <ListItemText primary="Hur länge innan föreställningen kan jag avboka min biljett?" />
         {open ? <ExpandLess /> : <ExpandMore />}
```

Comment:
i guess this codes you haven't finished yet?

Reply:
Yup, gonna fix it tomorrow :)

Reply:
took it away, have to sit down and solve that and it can take a while because of MUI not being co-operative with my plans. So i set the earlier version back for now.